src/components/hooks/useBranchInfo.js

```diff
@@ -12,15 +12,26 @@ export function useBranchInfo({ branchId }) {
       })
   );
 
-  const isBlocked =
-    userParams?.branches?.result?.[0]?.borrowerCheck !== false &&
-    !userParams?.branches?.borrowerStatus?.allowed;
+  // @TODO if agency is ffu .. and borrowerstatus is UNKNOWN_USER .. we simple need the pincode - no block please
+  const isBlocked = () => {
+    if (
+      userParams?.branches?.borrowerStatus?.statusCode === "UNKNOWN_USER" &&
+      userParams?.branches?.result?.[0]?.agencyType === "FORSKNINGSBIBLIOTEK"
+    ) {
+      return false;
+    } else {
+      return (
+        userParams?.branches?.result?.[0]?.borrowerCheck !== false &&
+        !userParams?.branches?.borrowerStatus?.allowed
+      );
+    }
+  };
 
   return {
     ...(userParams?.branches?.result?.[0] || {}),
     agencyUrl: userParams?.branches?.agencyUrl,
     branchId,
-    isBlocked,
+    isBlocked: isBlocked(),
```

Comment:
Skal vi have den her opdelt i 2 flag? eller bare have et flag for at vi ikke har haft mulighed for at kunne finde brugeren pga. manglende pinkode :shrug: Så vi ved jo egentlig ikke om han er blokeret eller ej. Bare en ide. Kan godt være det er overflødigt.